### DISCUSSION TITLE
Couldn't hide arrows for number inputs with the jsx parent `css` prop (using panda patterns)

### DISCUSSION THREAD

**Discussion by kiruushaaa** [chakra-ui/panda#2631] [Q&A] [answered]
The screenshot is from safari, but it actually isn't a reason.
Is it common behaviour or I'm missing something

playground: https://play.panda-css.com/bFO8yqcLqO

<img width="720" alt="image" src="https://github.com/chakra-ui/panda/assets/18754878/6c43c83f-c29c-41d8-8193-92be3e41c923">

**Comment by astahmer** [comment]
that's because you're scoping things in the `& input[type=number]` selector, whereas the `noArrows.raw()` is most likely designed to output CSS without a specific selector scope

**Reply by kiruushaaa** [reply]
so there is no processing for `&`, got it

are there any solutions that might fit this case?

**Reply by astahmer** [reply, marked as answer]
yes ! convert it to a [custom utility
](https://panda-css.com/docs/customization/utilities#using-boolean-values)https://play.panda-css.com/bFO8yqcLqO/vPY05QI-1q

**Reply by kiruushaaa** [reply]
don't understand how i didn't manage to find it through the docs 🤓
